### PR TITLE
FIX : Update main.inc.php

### PR DESCRIPTION
**Pull request by cmfpmatik**
Fix everywhere Warning error of PHP and log it in specific file.

### CODE REVIEW

**Comment by eldy** [comment]
Note that we already have a tool to track (differently) php warning: enable module debugbar.

**Comment by cmfpmatik** [comment]
> Note that we already have a tool to track (differently) php warning: enable module debugbar.

This fix is not a tool to track php warning, just to isn't displaying them. (safety recommendation)

**Comment by cmfpmatik** [comment]
what's happen about this ?

File: htdocs/main.inc.php [hunk 1]
```diff
@@ -36,6 +36,21 @@
...
+error_reporting(E_ALL ^ E_NOTICE);
+@set_error_handler("__error_handler");
+function __error_handler($errno, $errstr, $errfile, $errline)
+{
+	//echo "<pre><br/>ERREUR/WARNING : $errno, $errstr, $errfile, $errline<br/></pre>"; // for display it to user
+	trace("ERREUR/WARNING: $errno, $errstr, $errfile, $errline");
+}
+function trace($toTrace)
+{
+	$f = fopen("trace_warningphp.log", "a+");
```
Comment (eldy) [inline htdocs/main.inc.php:49]: On a secured installation, the application has no permission to write into a file into htdocs directory, so this won't work.

Also, if creating log, rotation of log must be managed.

Also hidding errors to users with no choice of user is not a good practice. user must choose if he want to not see important information he should see.

You don't need this to report errors onto a log file. You can already use the module "Log debug" but also the module "Debug bar".

**Pull request closed without merging**